external-crates/move/crates/move-compiler/src/editions/mod.rs

```diff
@@ -96,11 +96,19 @@ pub fn feature_edition_error_msg(edition: Edition, feature: FeatureGate) -> Opti
                     feature.error_prefix()
                 )
             } else {
-                format!(
-                    "{} not supported by current edition '{edition}', \
-                    only {} support this feature",
-                    feature.error_prefix(),
-                    format_oxford_list!("and", "'{}'", valid_editions)
+                valid_editions.last().map_or(
```

Comment:
I think we will want to work this as soon as there is a `202X.alpha` since all of these errors will start to change. 

If you want to change it now, I think the logic should be something like
- Is the feature supported by an edition without a `release`? If so, report the most recent such edition.
- Otherwise, is the feature supported by an edition with a `release`? If so, report the most recent such edition.
- Else, report the feature being in development.

Reply:
I think I'll punt on it in this PR, as I think there's some more generic refactoring around editions we may want to do around releases and alpha/beta versions of it (and this needs to land before the cut on Thursday). But agreed, that we will need to build out the logic here a bit to make this more resilient when adding new editions.